src/intervals/old_rationalize.jl

```diff
@@ -0,0 +1,29 @@
+# Version of `rationalize` from 0.3 that works correctly with rounding modes
```

Comment:
I think this comment should include a bit more details about the source file. Though the license is quite open, we should mention it somewhere. Or do you think this is worthless?